py/objmodule.c

```diff
@@ -173,7 +175,11 @@ static const mp_module_delegation_entry_t mp_builtin_module_delegation_table[] =
 // Attempts to find (and initialise) a built-in, otherwise returns
 // MP_OBJ_NULL.
 mp_obj_t mp_module_get_builtin(qstr module_name, bool extensible) {
```

Comment:
Maybe this function should have signature ` mp_obj_t mp_module_get_builtin(qstr module_name)` when extensible modules are disabled?  That might reduce code size a tiny bit more, be more obvious what's happening, and prevent unintended use of the `extensible` argument.